GL-README.md

```diff
@@ -58,11 +58,49 @@ After ~10 minutes, and then look in the `deploy/` for a file with a name like
 
 Copy this to your laptop, and then you can burn it to an SD card using the [Raspberry Pi Image](https://github.com/raspberrypi/rpi-imager).
 
-TODO: figure out how to use them inside `qemu`.
 TODO: set up some tests inside `qemu` that things are working.
 TODO: write those tests into CI/CD actions.
 
 
+## Running the Raspberry Pi Image with QEMU 
+To emulate the Raspberry Pi image with QEMU, we will first need to install the Linux kernel for QEMU on the 
+host machine and install other necessary packages required for QEMU to work. To do this, run 
+
+```shell 
+chmod +x setup-qemu-kernel.sh 
```

Comment:
You should chmod these scripts in the repo - git remembers these permission bits.